editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -178,7 +178,7 @@ describe('React Render Count Tests -', () => {
 
     const renderCountAfter = renderResult.getNumberOfRenders()
     // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`2382`)
+    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`581`)
```

Comment:
Wait what?! This is suspiciously good, but I guess intended...

Reply:
this number changed when I added flushSync. my theory that I didn't bother to verify is that what we see here is the effect of batching the updates. but I'm not too fussed to figure this out, because this test is more of a canary to flag PRs that did not intend to change this number and initiate a conversation

Reply:
Well yeah kinda, but if the number fluctuates by too much it's also worthwhile investigating to make sure that's not the result of a significant regression